src/android/io/sqlc/SQLitePlugin.java

```diff
@@ -216,7 +216,20 @@ private SQLiteAndroidDatabase openDatabase(String dbname, CallbackContext cbc, b
             Log.v("info", "Open sqlite db: " + dbfile.getAbsolutePath());
 
             SQLiteAndroidDatabase mydb = old_impl ? new SQLiteAndroidDatabase() : new SQLiteConnectorDatabase();
-            mydb.open(dbfile);
+            try {
+                mydb.open(dbfile);
```

Comment:
If `mydb` fails to open, then no cleanup code such as `close` or similar needs to be executed, right?

Reply:
Yep, that's right!